control-plane/subcommand/inject-connect/command.go

```diff
@@ -390,14 +392,19 @@ func (c *Command) Run(args []string) int {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		LeaderElection:         true,
-		LeaderElectionID:       "consul-controller-lock",
-		Host:                   listenSplits[0],
-		Port:                   port,
-		Logger:                 zapLogger,
-		MetricsBindAddress:     "0.0.0.0:9444",
+		Scheme:           scheme,
+		LeaderElection:   true,
+		LeaderElectionID: "consul-controller-lock",
+		Logger:           zapLogger,
+		Metrics: metricsserver.Options{
+			BindAddress: "0.0.0.0:9444",
+		},
 		HealthProbeBindAddress: "0.0.0.0:9445",
+		WebhookServer: webhook.NewServer(webhook.Options{
+			CertDir: c.flagCertDir,
+			Host:    listenSplits[0],
+			Port:    port,
+		}),
 	})
```

Comment:
Another breaking change in controller-runtime is that the webhook server gets initialized as part of the manager now -- including the `CertDir` -- instead of assigned in `v1controllers.go` and `v2controllers.go`.